components/elm/src/cpl/lnd_import_export.F90

```diff
@@ -1099,45 +1102,52 @@ subroutine lnd_import( bounds, x2l, atm2lnd_vars, glc2lnd_vars)
        atm2lnd_vars%forc_aer_grc(g,14) =  x2l(index_x2l_Faxa_dstdry4,i)
        
        !set the topounit-level atmospheric state and flux forcings
-       do topo = grc_pp%topi(g), grc_pp%topf(g)
-         ! first, all the state forcings
-         top_as%tbot(topo)    = x2l(index_x2l_Sa_tbot,i)      ! forc_txy  Atm state K
-         top_as%thbot(topo)   = x2l(index_x2l_Sa_ptem,i)      ! forc_thxy Atm state K
-         top_as%pbot(topo)    = x2l(index_x2l_Sa_pbot,i)      ! ptcmxy    Atm state Pa
-         top_as%qbot(topo)    = x2l(index_x2l_Sa_shum,i)      ! forc_qxy  Atm state kg/kg
-         top_as%ubot(topo)    = x2l(index_x2l_Sa_u,i)         ! forc_uxy  Atm state m/s
-         top_as%vbot(topo)    = x2l(index_x2l_Sa_v,i)         ! forc_vxy  Atm state m/s
-         top_as%zbot(topo)    = x2l(index_x2l_Sa_z,i)         ! zgcmxy    Atm state m
-         ! assign the state forcing fields derived from other inputs
-         ! Horizontal windspeed (m/s)
-         top_as%windbot(topo) = sqrt(top_as%ubot(topo)**2 + top_as%vbot(topo)**2)
-         ! Relative humidity (percent)
-         if (top_as%tbot(topo) > SHR_CONST_TKFRZ) then
+       !      call downscale_atmo_state_to_topounit(g, i, x2l)
+       if (use_downscaling_to_tpu) then
+	  !if (masterproc) then
+      !       write(iulog,*) 'use_downscaling_to_tpu = ', use_downscaling_to_tpu
+      !   endif
+         call downscale_grd_to_topounit(g, i, x2l, lnd2atm_vars)
+       else
```

Comment:
Remove commented code

Reply:
@thorntonpe I have removed commented code in my latest commit as requested. Thanks.